plugin/include/zeek-compat.h

```diff
@@ -56,7 +56,9 @@
 #include <zeek/analyzer/Analyzer.h>
 #include <zeek/analyzer/Manager.h>
 #include <zeek/analyzer/protocol/tcp/TCP.h>
+#if ZEEK_VERSION_NUMBER < 40100 // Zeek < 4.1
```

Comment:
Aren't we including this same file on l.82 below (we actually already seem to do this for `TCP.h`).

Unless this file goes away in newer version let's just include it unconditionally.

Reply:
it actually goes away

Reply:
Ah, missed how this added extra nesting to an existing `#if`.